### PR TITLE
Created userController

### PR DESCRIPTION
**Pull request by atsushii**
@bonnieli @ylee297 @mila-mamat @Adi-tya93 
Created userController

・calendarUrl Unique check
・update user information 
→ (name, avatarUrl, timeZone, availableHoursFrom, availableHoursTo, availableDays, calendarUrl)

### CODE REVIEW

File: server/validators/validateUserController.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  // Check that name is not null
+  if (!name) {
+    isValid = false;
+    message = "name shouldn't be an empty";
+    return { isValid, message };
+  }
+  // Check that is not null
+  if (!avatarUrl) {
+    isValid = false;
```
Comment (Adi-tya93) [inline server/validators/validateUserController.js:41]: The code 
if (!<field>)
 isValid = false;
 message = "<field> shouldn't be empty";
 return { isValid, message };
is repeated in some ways, you can combine it in a function like I did in appointmentController. This is not necessary.
I think the code looks good. One thing I would recommend is run it through postman to see if your controller is working for all scenarios and is bug free. I can help with that if you want.

File: server/validators/validateUserController.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    message = "avatarUrl shouldn't be an empty";
+    return { isValid, message };
+  }
+  // Check that timeZone is not null
```
Comment (Adi-tya93) [inline server/validators/validateUserController.js:45]: The code 
if (!<field>)
 isValid = false;
 message = "<field> shouldn't be empty";
 return { isValid, message };
is repeated in some ways, you can combine it in a function like I did in appointmentController. This is not necessary.
I think the code looks good. One thing I would recommend is run it through postman to see if your controller is working for all scenarios and is bug free. I can help with that if you want.

**Review by Adi-tya93 — COMMENTED** [review]
Some minor suggestions. If you are happy you can merge it.

File: server/validators/validateUserController.js (hunk 2 above)
Comment (atsushii) [reply server/validators/validateUserController.js:45]: Definitely, I don't need to write such a same code. Thank you for your exact advice! That is helpful :)
I checked my code using postman in my local env. I will check to connect with our DB.

Comment (Adi-tya93) [reply server/validators/validateUserController.js:45]: Happy to help. Our project is starting to take shape.

**Pull request merged**